docs/meshstack.authorization.md

```diff
@@ -164,10 +166,5 @@ The `MC_PROJECTS` claim contains all projects the user has access to in the scop
 
 #### Authorization via replication
 
-For platforms that don't support the [Authorization via OIDC](#authorization-via-oidc), access rights are replicated during project replication. Cloud platforms provide their own ACL system and meshStack configures it as defined in the meshProject. E.g. this could be an assignment of certain roles for a certain project in the cloud platform.
-
-## Role Revocation
-
-User role revocation on [meshProject](meshcloud.project.md#unassign-principal-from-a-meshproject) and [meshWorkspace](meshcloud.workspace.md#remove-assigned-meshworkspace-roles) level allow Workspace Managers to always limit access to the meshWorkspace and meshProjects to the users that actually need access. Users who no longer should have access can easily be revoked access. Administrators also have the possibility to revoke roles for a user to all meshWorkspaces and meshProjects and deactivate this user completely in the complete meshStack via the [delete user](administration.users.md#delete-user) functionality.
+For platforms that don't support the [Authorization via meshIdB](#meshidb-authorization), access rights are replicated during project replication. Cloud platforms provide their own ACL system and meshStack configures it as defined in the meshProject. E.g. this could be an assignment of certain roles for a certain project in the cloud platform.
```

Comment:
```suggestion
For platforms that don't support the [Authorization via meshIdB](#meshidb-authorization), access rights are replicated during project replication. Cloud platforms provide their own ACL system and meshStack configures it as defined in the project. E.g. this could be an assignment of certain roles for a certain project in the cloud platform.
```